server.go

```diff
@@ -33,6 +33,7 @@ type OsqueryPlugin interface {
 	Shutdown()
 }
 
+const defaultVersion = "0.0.0"
```

Comment:
I'm curious why set this to `0.0.0` and not leave it as the empty string?

Reply:
I also agree that I'd prefer to see empty as a default. Otherwise, this PR looks good to me. @cssmason if you make that change I'm happy to approve and merge.

Reply:
Does it take in an empty string?

Reply:
I think yes. Prior to this change, the default value (empty string) was used since the `Version` field was not set.

Reply:
@zwass 
I have set the default version as empty string. I am so glad to join this open source project to be a contributor.
Thanks a lot.

Reply:
Welcome aboard!